internal/schemautil/userconfig/apiconvert/toapi.go

```diff
@@ -0,0 +1,281 @@
+package apiconvert
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/aiven/terraform-provider-aiven/internal/schemautil/userconfig"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+var typedKeyRegexp = regexp.MustCompile(`^[A-z]*_(boolean|integer|number|string|array|object)$`)
+
+// arrayItemToAPI is a function that converts array property of Terraform user configuration schema to API
+// compatible format.
+func arrayItemToAPI(
+	n string,
+	fk []string,
+	k string,
+	v []interface{},
+	i map[string]interface{},
+	d *schema.ResourceData,
+) (interface{}, bool) {
+	var res = make([]interface{}, len(v))
+
+	fks := strings.Join(fk, ".")
+
+	// TODO: Remove when this is fixed on backend.
+	if k == "additional_backup_regions" {
+		return res, true
+	}
```

Comment:
What fix it waits for? What `true` stands for?

Reply:
it waits for the fix of `additional_backup_regions`, `true` stands for omit